dashboard/modules/reporter/tests/test_reporter.py

```diff
@@ -697,5 +698,63 @@ def _generate_worker_key(self, proc):
             agent_mock.kill()
 
 
+TASK = {
+    "task_id": "32d950ec0ccf9d2affffffffffffffffffffffff01000000",
+    "attempt_number": 0,
+}
+
+
+def test_get_task_traceback():
+    """
+    Verify that we throw an error for a non-running task.
+    """
+    context = ray.init()
+    dashboard_url = f"http://{context['webui_url']}"
+
+    @ray.remote
+    def f():
+        pass
+
+    ray.get([f.remote() for _ in range(5)])
+
+    # Make sure the API works.
+    def verify():
+        with pytest.raises(requests.exceptions.HTTPError) as exc_info:
+            resp = requests.get(
+                f"{dashboard_url}/task/traceback?task_id={TASK['task_id']}&attempt_number={TASK['attempt_number']}"
+            )
+            resp.raise_for_status()
+        assert isinstance(exc_info.value, requests.exceptions.HTTPError)
```

Comment:
so we never test if it works in unit test?

Reply:
I will add two test cases for happy path